src/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkImageBrowser.tsx

```diff
@@ -49,7 +49,7 @@ export class LargeArtworkImageBrowser extends React.Component<
     }
 
     return (
-      <Container>
+      <Container key={Math.random()}>
```

Comment:
Safety mechanism to get around mount / unmount of flickity carousel, as commented on below. I don't have a good solution yet after a day of debugging. But setting a new key, the dom node reinitializes, getting around the problem. It's not noticeable in the app.